Assets/Scripts/SS3D/Systems/Chat/InGameChatController.cs

```diff
@@ -0,0 +1,122 @@
+﻿using FishNet.Object;
+using FishNet.Object.Synchronizing;
+using SS3D.Core;
+using SS3D.Core.Behaviours;
+using SS3D.Systems.Inventory.Containers;
+using SS3D.Systems.Inventory.Items;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+namespace SS3D.Engine.Chat
+{
+    public sealed class InGameChatController : NetworkActor
```

Comment:
Why ingame? why not ChatController?

Reply:
This one is specifically for the InGameChatWindow, part of a Human GameObject, so basically is the link between InGameChatWindow and Human